composer/trainer/trainer.py

```diff
@@ -1443,7 +1443,7 @@ def __init__(
                 # Set load arguments to defaults as this applies only to the initial non-autoresume
                 # load. We do not reset `load_strict_model_weights` for models with frozen layers.
                 load_weights_only = False
-                load_ignore_keys = None
+                # load_ignore_keys = None  # TODO: Re-enable load ignore keys
```

Comment:
reminder to undo

Reply:
Leaving for @abhi-mosaic. I have a reminder to revert pre release